### PR TITLE
Update part2.md

### PR DESCRIPTION
**Pull request by Mesqualito**
Maybe you can describe it in better words than mine.
Or maybe it is a local or Cmder problem only at my side?

When I "quit" the docker-container with CTRL+C, the container stays alive, which can be controlled with docker ps or with browsing "http://localhost:4000". When I try to do the next command in the documentation ("docker run -d -p 4000:80 friendlyhello"), there will be an error: "Port is already allocated". So additionally to CTRL+C I have to stop the container before I can continue.

( Windows 10 Pro N, Docker version 17.03.1-ce, build c6d412e (Docker for Windows), Terminal: Cmder/ConEmu 161022 [64] {Stable} )

### CODE REVIEW

**Comment by mstanleyjones** [comment]
@thaJeztah is this expected behavior?

**Comment by thaJeztah** [comment]
Hm, no this is not expected; at least it's different than the behaviour on Linux / OS X; pressing CTRL+C sends a terminate signal to the container's process, and stops the container. Possibly an issue with ConEMU?

I don't have access to a Windows machine, but perhaps @johnstep or @friism can verify

File: get-started/part2.md [hunk 1]
```diff
@@ -202,7 +202,7 @@ between what you `EXPOSE` within the `Dockerfile`, and what you `publish` using
 `docker run -p`. In later steps, we'll just map port 80 on the host to port 80
 in the container and use `http://localhost`.
 
-Hit `CTRL+C` in your terminal to quit.
+Hit `CTRL+C` in your terminal to quit STDOUT from the container and run `docker ps` to get the container id to stop it with `docker stop [container id]`.
```
Comment (mstanleyjones) [inline get-started/part2.md]: I don't think "quitting STDOUT" is a thing. Is there a way to rephrase this to be more accurate?

Comment (JimGalasyn) [reply get-started/part2.md]: I think this behavior is specific the cmder shell, so I'm not inclined to add this special case to the topic.

**Comment by johndmulhausen** [comment]
Closing per Jim's last comment. Thanks all!
